app/src/main/java/emu/skyline/MainActivity.kt

```diff
@@ -120,15 +122,25 @@ class MainActivity : AppCompatActivity() {
         binding.swipeRefreshLayout.apply {
             setProgressBackgroundColorSchemeColor(obtainStyledAttributes(intArrayOf(R.attr.colorPrimary)).use { it.getColor(0, Color.BLACK) })
             setColorSchemeColors(obtainStyledAttributes(intArrayOf(R.attr.colorAccent)).use { it.getColor(0, Color.BLACK) })
-            post { setDistanceToTriggerSync((binding.swipeRefreshLayout.height / 2.5f).roundToInt()) }
+            post { setDistanceToTriggerSync(binding.swipeRefreshLayout.height / 3) }
             setOnRefreshListener { loadRoms(false) }
         }
 
         viewModel.stateData.observe(this, ::handleState)
         loadRoms(!settings.refreshRequired)
 
         binding.searchBar.apply {
-            binding.logIcon.setOnClickListener { startActivity(Intent(context, LogActivity::class.java)) }
+            binding.logIcon.setOnClickListener {
```

Comment:
I think this would look better if it were written in ☕ 🧻

Reply:
Two people disagree so you're clearly wrong here, it looks the best in Kotlin :sunglasses: